Configuration/odin-app.json

```diff
@@ -15,6 +15,9 @@
                         "expires": {"hours": 72},
                         "static": "usr/share/odin/Static/app/login.html"
                     }
+                },
+                "verify/": {
```

Comment:
This name looks wrong. Whouldn't there be `views/` at the front of it?

Reply:
It's a part of `pathprefix` view configuration, not views